v2/blockstore/insertionindex.go

```diff
@@ -1,29 +1,36 @@
-package index
+package blockstore
 
 import (
 	"bytes"
 	"encoding/binary"
+	"errors"
 	"fmt"
 	"io"
 
+	"github.com/ipld/go-car/v2/index"
+	"github.com/multiformats/go-multicodec"
+
 	"github.com/ipfs/go-cid"
 	"github.com/multiformats/go-multihash"
 	"github.com/petar/GoLLRB/llrb"
 	cbor "github.com/whyrusleeping/cbor/go"
 )
 
-type InsertionIndex struct {
-	items llrb.LLRB
-}
+var (
+	errUnsupported      = errors.New("not supported")
+	insertionIndexCodec = 0x300003
```

Comment:
surely this can be replaced by go-multicodec now?

Reply:
this is the insertion index codec. Do we have a codec for that? I thought we only had one for sorted index.

Reply:
Replaced this specific constant with a wrapped `multicodec.Code`. Please let me know if I missed anything. thanks

Reply:
Ah of course, I had misread the name. SGTM